<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoPage</class>
 <widget class="QWidget" name="InfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>399</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Page settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>Paper size:</string>
        </property>
        <property name="buddy">
         <cstring>CmbPaperSize</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="CmbPaperSize">
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>22</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>A4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Letter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10x15cm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13x17cm</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="m_fullbleed">
        <property name="text">
         <string>no margins</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Captions:</string>
        </property>
        <property name="buddy">
         <cstring>m_captions</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="m_captions">
          <item>
           <property name="text">
            <string>No captions</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Image file names</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Exif date-time</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Comments</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Free</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="m_FreeCaptionFormat">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="m_free_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>%f  filename, %t exposure time, %c comment, %i ISO, %d date-time, %r resolution, %a aperture, %l focal length, \n newline</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Family:</string>
        </property>
        <property name="buddy">
         <cstring>m_font_name</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QFontComboBox" name="m_font_name"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Color:</string>
        </property>
        <property name="buddy">
         <cstring>m_font_color</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="KColorButton" name="m_font_color">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="textLabel1_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Size:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>m_font_size</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="m_font_size">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GrpOutputSettings">
     <property name="title">
      <string>Output Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="RdoOutputGimp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Open image using GIMP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RdoOutputPrinter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Output to printer</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RdoOutputFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Output to image file</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="EditOutputPath">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnBrowseOutputPath">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_outfile">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Each page is named kipi_printassistant_N.jpeg (N is 1, 2, etc.) and saved to this folder.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>GrpOutputSettings</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>CmbPaperSize</tabstop>
  <tabstop>m_fullbleed</tabstop>
  <tabstop>m_captions</tabstop>
  <tabstop>m_FreeCaptionFormat</tabstop>
  <tabstop>m_font_name</tabstop>
  <tabstop>m_font_color</tabstop>
  <tabstop>m_font_size</tabstop>
  <tabstop>RdoOutputGimp</tabstop>
  <tabstop>RdoOutputPrinter</tabstop>
  <tabstop>RdoOutputFile</tabstop>
  <tabstop>EditOutputPath</tabstop>
  <tabstop>BtnBrowseOutputPath</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>RdoOutputFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditOutputPath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RdoOutputFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_outfile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RdoOutputFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>BtnBrowseOutputPath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
